Dockerfile

```diff
@@ -47,6 +47,8 @@ RUN npm ci --ignore-scripts --only=production
 
 FROM base AS app
 
+VOLUME "/tmp/" "/app/tmp/sockets/"
```

Comment:
Why not mount the whole `/app/tmp` folder? 🤔

Reply:
iirc I tried that and it complained it couldn't write to `/app/tmp/sockets` on ECS

Reply:
How strange... So we just have to be aware that if in future Rails wants to write to other paths in `tmp`, we'll have to add those separately? Might be worth adding a comment about this...

Reply:
Yes, I think that's right. I'll add the comment.